.github/workflows/symbolicator-integration-test.yml

```diff
@@ -0,0 +1,58 @@
+name: symbolicator integration
+on:
+  push:
+    branches:
+      - master
+      - releases/**
+  pull_request:
+
+jobs:
+  test:
+    name: symbolicator test
```

Comment:
No `if: steps.changes.outputs.frontend == 'false'` for this job's steps?

Reply:
Oop, good catch, thanks